src/api/callback.cc

```diff
@@ -130,7 +134,7 @@ void InternalCallbackScope::Close() {
     return;
   }
 
-  HandleScope handle_scope(env_->isolate());
+  HandleScope handle_scope(isolate);
```

Comment:
hm, I know this exists before this PR, but why do we create a handle scope here? May be it should just be deleted, AFAICT this is meant to be invoked when there is already a HandleScope

Reply:
https://github.com/nodejs/node/commit/bca23f2ee71ddaf4f4393652de75dbb7fbc27f1d

/cc @addaleax

Reply:
I mean, just because there’s an outer handle scope doesn’t mean that it’s pointless to have an inner one, e.g. when `InternalCallbackScope` is used in a loop